todoApp/spec/tspconfig.yaml

```diff
@@ -1,9 +1,12 @@
 emit:
+ - "@typespec/openapi3"
  - "@typespec/http-server-csharp"
- - "@typespec/http-server-javascript"
  - "@typespec/http-client-csharp"
  - "@typespec/http-client-java"
+ - "@typespec/http-server-javascript"
 options:
+  "@typespec/openapi3":
+    emitter-output-dir: "{project-root}/../openapi3"
```

Comment:
A side question, what is block us from merge js client emitter options here? @v-jiaodi @MaryGao

Reply:
If MFD is not ready, let's try to avoid the crash of CodeGen for ToDo project so that we could still generate the basic functions that could verify other E2E scenarios for JS.